.changeset/afraid-months-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@builder.io/qwik-city': patch
+'@builder.io/qwik': patch
+---
+
+Fix error boundary
```

Comment:
we should use `fix: ` prefix for changes
Maybe we could also inform users more about changes?